packages/fxa-payments-server/src/lib/amplitude.ts

```diff
@@ -160,31 +160,27 @@ export function createSubscription_REJECTED(eventProperties: EventProperties) {
   );
 }
 
-export function updatePaymentMounted(eventProperties: EventProperties) {
-  safeLogAmplitudeEvent(
-    eventGroupNames.updatePayment,
-    eventTypeNames.view,
-    eventProperties
-  );
+export function updatePaymentMounted(eventProperties: EventProperties = {}) {
```

Comment:
I waffled a bit on this since Amplitude event properties can be any key-value pairs.  But since we are not really using the param, it might be confusing to leave it there.  I'll remove it.